rest-service/manager_rest/resource_manager.py

```diff
@@ -1022,6 +1025,8 @@ def prepare_executions(self, executions, *, force=False, queue=False,
             executions.extend(component_executions)
         if commit:
             db.session.commit()
+        if errors:
+            raise errors[0]
```

Comment:
Do we already have a jira for making it raise a list (at least in user, if not machine, readable form)?

Reply:
This is RD-3166